src/ui/components/SpecialDistribution.js

```diff
@@ -1,38 +1,73 @@
 // @flow
-
 import React, {useState} from "react";
-import {div, fromInteger, fromFloatString, ZERO} from "../../ledger/grain";
-import {type IdentityId} from "../../ledger/identity";
+import {Button, Container, TextField} from "@material-ui/core";
+import {Alert} from "@material-ui/lab";
+import {makeStyles} from "@material-ui/core/styles";
+import {div, fromFloatString, lt, ZERO} from "../../ledger/grain";
 import {type Account} from "../../ledger/ledger";
 import AccountDropdown from "./AccountSelector";
 import {computeAllocation} from "../../ledger/grainAllocation";
 import * as uuid from "../../util/uuid";
 import type {TimestampMs} from "../../util/timestamp";
 import {useLedger} from "../utils/LedgerContext";
 
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "80%",
+    minWidth: "1100px",
```

Comment:
Seems pretty high for a minWidth. IMO should only set the minWidth of specific components if they need to have a minWidth set. Root level minwidth just makes it hard to support smaller screens.

Reply:
set to 700px instead. Any smaller than this and it really starts to look terrible without a minimum width.